README.md

```diff
@@ -1,19 +1,20 @@
-# Membrane Multimedia Framework: Mad Element
+# Membrane MP3 MAD plugin
 
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane-element-mad.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane-element-mad)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_mp3_mad_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_mp3_mad_plugin)
 
-This package provides [Membrane Multimedia Framework](https://membraneframework.org)
-elements that can be used to decode MPEG audio using MAD library.
+MP3 decoder based on MAD.
 
-Documentation is available at [HexDocs](https://hexdocs.pm/membrane_element_mad/)
+This package is a part of [Membrane Multimedia Framework](https://membraneframework.org).
+
+Documentation is available at [HexDocs](https://hexdocs.pm/membrane_mp3_mad_plugin/)
```

Comment:
Maybe we could turn this into badges like in Unifex but I don't insist.

Reply:
Haven't noticed that comment and merged :P So maybe next time ;)